hw/Makefile

```diff
@@ -109,17 +115,28 @@ $(ips_reg): %_reg: | $(REG_OUTPUT_DV_DIR)
 	${PRJ_DIR}/util/regtool.py --sec-cm-testplan $(blk-hjson)
 	${PRJ_DIR}/util/regtool.py -s -t $(REG_OUTPUT_DV_DIR) $(blk-hjson)
 
+# Unsecure, entropy buffer file generation for use by `topgen.py`,
+# `gen-otp-mmap.py` and `gen-lc-state-enc.py` scripts.
+#
+# BEFORE FINAL TAPEOUT: replace this rule with one that generates secure entropy
+# buffer files in the same directory locations.
+.PHONY: entropy-buffer-files-unsecure
+entropy-buffer-files-unsecure:
+	${PRJ_DIR}/util/topgen/entropy_buffer_generator.py -n 20000 --seed 4881560218908238235 -o $(TOPGEN_ENTROPY_BUFFER) && \
+	${PRJ_DIR}/util/topgen/entropy_buffer_generator.py -n 20000 --seed 10167336684108184581 -o $(LC_STATE_ENC_ENTROPY_BUFFER) && \
```

Comment:
I think that by changing this to `10167336684108184581 + LC_SEED_DIVERSIFIER`, we can avoid updating auto-generated files.

In the current flow strong_random.buffer is filled by:
`strong_random.unsecure_generate_from_seed(
                ENTROPY_BUFFER_SIZE_BYTES,
                LC_SEED_DIVERSIFIER + int(config['seed']))`
If we generate a file using entropy_buffer_generator with seed `LC_SEED_DIVERSIFIER + int(config['seed']` and load strong.random buffer by: 
`strong_random.load(config['entropy_buffer'])`, the generated random numbers should be the same and none of the autogenerated files would be changed.

This might end up being necessary if some of these files are already locked by RTL freeze.

Reply:
Ah right, I forgot to move over the diversifier to the Makefile, let me fix that. Good catch, thanks!